docs/book/src/tasks/experimental-features/cluster-resource-set.md

```diff
@@ -8,9 +8,44 @@ The `ClusterResourceSet` feature is introduced to provide a way to automatically
 
 The `ClusterResourceSet` feature is enabled by default, but can be disabled by setting the `EXP_CLUSTER_RESOURCE_SET` environment variable to `false`.
 
-More details on `ClusterResourceSet` and an example to test it can be found at:
+More details on `ClusterResourceSet` can be found at:
 [ClusterResourceSet CAEP](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20200220-cluster-resource-set.md)
 
+## Example
+
+Suppose you want to automatically install the relevant external cloud provider on all workload clusters.
+This can be accomplished by labeling the clusters with the specific cloud (e.g. AWS, GCP or OpenStack) and then create a `ClusterResourceSet` for each.
```

Comment:
```suggestion
This can be accomplished by labeling the clusters with the specific cloud (e.g. AWS, GCP or OpenStack) and then creating a `ClusterResourceSet` for each.
```